shell/client/shell.html

```diff
@@ -546,6 +546,11 @@ <h4 class="who-has-access">Who has access</h4>
     </div>
     {{#if completionState.clear}}
     <div class="button-container" role="presentation">
+      {{#with invitationExplanation}}
```

Comment:
This is a newer Spacebars feature, so we're far from reliably using it in the rest of the codebase, but a clearer way to write this would use the second syntax at https://github.com/meteor/meteor/blob/devel/packages/spacebars/README.md#with where you bind a name to that value, rather than hoisting the whole thing into your current data context.

```
{{#with explanation=invitationExplanation}}
<p class="invitation-explanation" title="{{explanation}}">{{explanation}}</p>
{{/with}}
```